app/models/host/managed.rb

```diff
@@ -2,7 +2,12 @@ class Host::Managed < Host::Base
   include Hostext::Search
   include SelectiveClone
   include HostParams
-  PROVISION_METHODS = %w[build image]
+
+  class_attribute :provision_methods
```

Comment:
To go with my comment about accessing the registered methods from the list of plugins, you could then skip the class attr and in the getter simply return `{'build'...}.merge(registered_methods)`.